community/examples/ml-gke.yaml

```diff
@@ -22,10 +22,13 @@ vars:
   zones:
   - asia-southeast1-b  # g2 machine has better availability in this zone
 
-  # Cidr block containing the IP of the machine calling terraform.
-  # The following line must be updated for this example to work.
+  # Uncomment the following lines if you want to add
+  # cidr block containing the IP of the machine calling terraform.
   authorized_cidr: <your-ip-address>/32
```

Comment:
new comment above says "uncomment" but the authorized cidr is not commented out.

Reply:
Updated and removed the changes to blueprint.